<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcWindow</class>
 <widget class="QMainWindow" name="CalcWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
        background-color: &quot;#363636&quot;;
}

QPushButton {
        border-radius: 8px;
        color: white;
        background-color: grey;
        font-size: 18px;
        font-weight: bold;
}

QPushButton:hover:!pressed {
        border: 4px solid white;
}

QLineEdit {
        border: 0px;
        border-radius: 8px;
        background-color: transparent;
        font-size: 32px;
        color: white;
        font-weight: bold;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
    <item>
     <widget class="QLineEdit" name="editCalc">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editOut">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="xLabel">
        <property name="styleSheet">
         <string notr="true">QLabel {
        color: white;
        font-size: 18px;
        font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="xEdit">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
        border: 1px solid white;
        border-radius: 8px;
        background-color: transparent;
        font-size: 18px;
		padding: 2px;
        color: white;
        font-weight: bold;
}</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="4">
       <widget class="QPushButton" name="n9_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="7" column="5">
       <widget class="QPushButton" name="divBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#4287f5&quot; }

QPushButton:pressed { background-color: &quot;#aa4287f5&quot; }</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="asinBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#32a852&quot; }

QPushButton:pressed { background-color: &quot;#aa32a852&quot; }</string>
        </property>
        <property name="text">
         <string>asin</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="acosBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#32a852&quot; }

QPushButton:pressed { background-color: &quot;#aa32a852&quot; }</string>
        </property>
        <property name="text">
         <string>acos</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="dotBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: &quot;#7700ff&quot; }

QPushButton:pressed { background-color: &quot;#aa7700ff&quot; }</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="powBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#fcba03&quot; }

QPushButton:pressed { background-color: &quot;#aafcba03&quot; }</string>
        </property>
        <property name="text">
         <string>pow</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="mulBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#4287f5&quot; }

QPushButton:pressed { background-color: &quot;#aa4287f5&quot; }</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="n4_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="n8_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="logBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#fcba03&quot; }

QPushButton:pressed { background-color: &quot;#aafcba03&quot; }</string>
        </property>
        <property name="text">
         <string>log</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="n2_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="rmBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#f54242&quot;; }

QPushButton:pressed { background-color: &quot;#aaf54242&quot; }</string>
        </property>
        <property name="text">
         <string>✖️</string>
        </property>
       </widget>
      </item>
      <item row="7" column="4">
       <widget class="QPushButton" name="computeBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#7700ff&quot; }

QPushButton:pressed { background-color: &quot;#aa7700ff&quot; }</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="closeBktBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: &quot;#7700ff&quot; }

QPushButton:pressed { background-color: &quot;#aa7700ff&quot; }</string>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="openBktBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: &quot;#7700ff&quot; }

QPushButton:pressed { background-color: &quot;#aa7700ff&quot; }</string>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="lnBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#fcba03&quot; }

QPushButton:pressed { background-color: &quot;#aafcba03&quot; }</string>
        </property>
        <property name="text">
         <string>ln</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="n6_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="n3_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="n7_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="xBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: &quot;#7700ff&quot; }

QPushButton:pressed { background-color: &quot;#aa7700ff&quot; }</string>
        </property>
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="n1_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="cosBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#32a852&quot; }

QPushButton:pressed { background-color: &quot;#aa32a852&quot; }</string>
        </property>
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="sinBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#32a852&quot; }

QPushButton:pressed { background-color: &quot;#aa32a852&quot; }</string>
        </property>
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QPushButton" name="minusBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#4287f5&quot; }

QPushButton:pressed { background-color: &quot;#aa4287f5&quot; }</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="plusBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#4287f5&quot; }

QPushButton:pressed { background-color: &quot;#aa4287f5&quot; }</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="n5_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="atanBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#32a852&quot; }

QPushButton:pressed { background-color: &quot;#aa32a852&quot; }</string>
        </property>
        <property name="text">
         <string>atan</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="modBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#fcba03&quot; }

QPushButton:pressed { background-color: &quot;#aafcba03&quot; }</string>
        </property>
        <property name="text">
         <string>mod</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="acBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string>QPushButton { background-color: &quot;#f54242&quot;; }

QPushButton:pressed { background-color: &quot;#aaf54242&quot; }</string>
        </property>
        <property name="text">
         <string>AC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="sqrtBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#fcba03&quot; }

QPushButton:pressed { background-color: &quot;#aafcba03&quot; }</string>
        </property>
        <property name="text">
         <string>sqrt</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="tanBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#32a852&quot; }

QPushButton:pressed { background-color: &quot;#aa32a852&quot; }</string>
        </property>
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="n0_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#a2a2d0&quot; }

QPushButton:pressed { background-color: &quot;#aaa2a2d0&quot; }</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="6">
       <widget class="QPushButton" name="drawBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton { background-color: &quot;#9dbeb9&quot; }

QPushButton:pressed { background-color: &quot;#aa9dbeb9&quot; }</string>
        </property>
        <property name="text">
         <string>Draw</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="modesBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>698</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModes">
    <property name="title">
     <string>Modes</string>
    </property>
    <addaction name="actionCredit"/>
    <addaction name="actionDeposit"/>
   </widget>
   <addaction name="menuModes"/>
  </widget>
  <action name="actionCredit">
   <property name="text">
    <string>Бизнес калькулятор ($_$)</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Калькулятор доходности вкладов</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
